shopfloor_mobile/static/wms/src/scenario/zone_picking.js

```diff
@@ -284,7 +290,11 @@ const ZonePicking = {
             }
             const data = this.state_get_data("select_line");
             if (_.isEmpty(data) || _.isEmpty(data.picking_type)) {
-                return {};
+                const data_sl = this.state_get_data("scan_location");
+                if (_.isEmpty(data_sl) || _.isEmpty(data_sl.buffer_is_loaded)) {
+                    return {};
+                }
+                return data_sl.buffer_is_loaded.picking_type;
```

Comment:
```suggestion
                const buffer = this.state_get_data("scan_location").buffer;
                if (_.isEmpty(buffer)) {
                    return {};
                }
                return buffer.picking_type;
```
same for zone_location ;)

Reply:
done